tck/src/main/java/org/eclipse/microprofile/rest/client/tck/cditests/CDIInvokeWithRegisteredProvidersTest.java

```diff
@@ -108,7 +108,7 @@ public void testInvokesPutOperation() throws Exception {
             .willReturn(aResponse()
                 .withBody(outputBody)));
 
-        Response response = api.executePut(id, inputBody);
+        Response response = api.executePut(expectedId, inputBody);
```

Comment:
This test checks that the providers are invoked, not the server behavior.

Reply:
See my comment above